lib/nethealth/nethealth.go

```diff
@@ -240,6 +244,18 @@ func (s *Server) Start() error {
 			s.Fatalf("ListenAndServe(): %s", err)
 		}
 	}()
+	go func() {
+		_ = os.Remove(DefaultNethealthSocket)
```

Comment:
L248-L255 (everything up to httpServer.Serve should be executed outside of the goroutine as it does not make sense to continue upon these failures. This will eliminate the need to panic